viewer/lib/utils.js

```diff
@@ -22,6 +27,9 @@ function loadTexture (texture, cb) {
 }
 
 function loadJSON (json, cb) {
+  if (process.platform === 'browser') {
+    return require('./utils.web').loadJSON(json, cb)
```

Comment:
why is this needed? didn't it work before?

Reply:
As said in the body I moved to esbuild in my p viewer setup so I guess that's why I added it. If this is the only thing that blocks merging the pr and you don't want to see it you can remove it (feel free to do it since you should have push access)